MapboxNavigation/RouteMapViewController.swift

```diff
@@ -205,6 +205,15 @@ class RouteMapViewController: UIViewController, PulleyPrimaryContentControllerDe
         guard let controller = routePageViewController.currentManeuverPage else { return }
         
         controller.notifyDidChange(routeProgress: routeProgress, secondsRemaining: secondsRemaining)
+        updateShield(for: controller)
+        
+        // Move the overview button if the lane views become visible
+        if !controller.isPagingThroughStepList {
+            let initialPaddingForOverviewButton:CGFloat = controller.stackViewContainer.isHidden ? -30 : -20 + controller.laneViews.first!.frame.maxY
```

Comment:
It's just because the lane view is out of bounds. We should be able to remove all this but it requires some tweaks in the storyboard. Let's do that in a separate PR.